README.md

```diff
@@ -399,6 +399,10 @@ Set up your GitHub Actions workflow with a specific version of your programming
 
 - [Sync/upload a directory to an AWS S3 bucket](https://github.com/jakejarvis/s3-sync-action)
 - [Deploy Lambda code to an existing function](https://github.com/appleboy/lambda-action)
+- [Run awscli](https://github.com/clowdhaus/aws-github-actions)
```

Comment:
Since these all reference the same repo, and is referenced as a collection of actions, would you mind moving this to the bottom of the "Collection of Actions" section?

Reply:
Also, please be mindful that all titles should be Title Cased: https://github.com/sdras/awesome-actions/blob/master/contributing.md